deepchecks/recommender/checks/sample_performance.py

```diff
@@ -59,24 +54,29 @@ def __init__(self,
 
     def run_logic(self, context: Context, dataset_kind) -> CheckResult:
         """Run check."""
+        if self.n_samples ==None:
+            dataset = context.get_data_by_kind(dataset_kind)
+
         dataset = context.get_data_by_kind(dataset_kind).sample(self.n_samples, random_state=self.random_state)
         model = context.model
-        scorers = context.get_scorers(self.scorers, use_avg_defaults=False)
+        scorers = context.get_scorers(self.scorers, use_avg_defaults=True)
 
         results = []
         display = None
         for scorer in scorers:
             scorer_value = scorer(model, dataset)
             results.append([scorer.name, scorer_value])
         results_df = pd.DataFrame(results, columns=['Metric', 'Value'])
+        
         if context.with_display:
-            fig = go.Figure()
-            for _, row in results_df.iterrows():
-                box_fig = go.Box(y=row['Value'], name=row['Metric'])
-                fig.add_trace(box_fig)
-            display = fig
+            fig = px.bar(results_df,y='Value', x='Metric')
+            
+        text = '''
+                The metrics are computed over {:,.0f} samples.
```

Comment:
This should be a footnote if exists at all, look at footnotes in other checks

Reply:
Fixed. 20b60a9ce699979ae778dcba17b6c9cc5a8cbff5
<img width="855" alt="Capture d’écran 2023-07-22 à 12 32 31 PM" src="https://github.com/deepchecks/deepchecks/assets/55285736/bfb9af3e-4fe9-4b98-9c7f-303f1eb165b7">